dev/ledger.md

```diff
@@ -820,13 +820,18 @@ and contains the following fields:
     of the entries follows the execution order of the `log`
     invocations. The maximum total number of `log` calls is 32, and the
     total size of all logged bytes is limited to 1024.
-  - Zero or more `InnerTxns`, encoded in an array `itx`. Each element of
-    `itx` records a successful invocation of the `tx_submit` opcode. Each
-    element will contain the transaction fields, encoded under `txn`, in
-    the same way that the top-level transaction is encoded, recursively,
-    including `ApplyData` that applies to the inner transaction.  Up to 16
-    `InnerTxns` may be present.
-    - InnerTxns are limited to `pay`, `axfer`, `acfg`, and `afrz` transactions.
+  - Zero or more `InnerTxns`, encoded in an array `itx`. Each element
+    of `itx` records a successful inner transaction. Each element will
+    contain the transaction fields, encoded under `txn`, in the same
+    way that the top-level transaction is encoded, recursively,
+    including `ApplyData` that applies to the inner transaction.
+    - Up to 16 `InnerTxns` may be present in version 5. In version 6,
+    the count of all inner transactions across the transaction group
+    must not exceed 16 $*$ the number of top-level ApplicationCall
```

Comment:
i thought we could have one app transaction that then did up to 256 inner txes in Teal 6.

Reply:
Yes, that was a recent decision. Will update.